code/datums/autolathe/arms.dm

```diff
@@ -137,44 +137,31 @@
 	path =/obj/item/ammo_magazine/m44
 	hidden = 1
 
-/////// 5.56mm
-/datum/category_item/autolathe/arms/rifle_556
-	name = "rifle magazine (5.56mm)"
-	path =/obj/item/ammo_magazine/m556
+/////// 5.45mm
+/datum/category_item/autolathe/arms/rifle_545
+	name = "rifle magazine (5.45mm)"
+	path =/obj/item/ammo_magazine/m545
 	hidden = 1
 
-/datum/category_item/autolathe/arms/rifle_556p
-	name = "rifle magazine (5.56mm practice)"
-	path =/obj/item/ammo_magazine/m556/practice
+/datum/category_item/autolathe/arms/rifle_545p
+	name = "rifle magazine (5.45mm practice)"
+	path =/obj/item/ammo_magazine/m545/practice
 
-/datum/category_item/autolathe/arms/machinegun_556
+/datum/category_item/autolathe/arms/machinegun_545
 	name = "machinegun box magazine (5.56)"
-	path =/obj/item/ammo_magazine/m556saw
+	path =/obj/item/ammo_magazine/m545saw
 	hidden = 1
-
 /////// 7.62
 
+
 /datum/category_item/autolathe/arms/rifle_762
 	name = "rifle magazine (7.62mm)"
 	path =/obj/item/ammo_magazine/m762
 	hidden = 1
-/*
-/datum/category_item/autolathe/arms/rifle_small_762
-	name = "rifle magazine (7.62mm)"
-	path =/obj/item/ammo_magazine/s762
-	hidden = 1
-*/
-
-/////// Shotgun
-
-/datum/category_item/autolathe/arms/shotgun_clip_beanbag
-	name = "4-round 12g shell clip (beanbag)"
-	path =/obj/item/ammo_magazine/clip/c12g/beanbag
 
 /datum/category_item/autolathe/arms/shotgun_clip_slug
 	name = "4-round 12g shell clip (slug)"
 	path =/obj/item/ammo_magazine/clip/c12g
-	hidden = 1
```

Comment:
The fuck? I didn't change that. Shit, do I have an outdated version of the code? God damn it, STARTING AGAIN.